src/rSharp.cpp

```diff
@@ -50,10 +53,30 @@ void rSharp_create_domain() {
 	}
 
 	// STEP 2: Initialize and start the .NET Core runtime
-	const string_t config_path = STR("./RSharp.runtimeconfig.json");
 	load_assembly_and_get_function_pointer = nullptr;
-	load_assembly_and_get_function_pointer = get_dotnet_load_assembly(config_path.c_str());
+	load_assembly_and_get_function_pointer = get_dotnet_load_assembly(wideStringPath);
 	assert(load_assembly_and_get_function_pointer != nullptr && "Failure: get_dotnet_load_assembly()");
+
+	delete[] wideStringPath;
+}
+
+char_t* MergeLibPath(char** libPath, char* additionalPath)
+{
+	size_t libPathLen = strlen(*libPath);

```

Comment:
This is another one of those situations where there's lots of cleanup to do, but let's start eliminating abbreviations from variable and function/method names.

Reply:
It's much more clear for me to read ```libararyPathLength``` or ```library_path_length```